### PR TITLE
Get user insights

### PR DESCRIPTION
**Pull request by kevinjon27**
New endpoint:
 `insights/account_organic_insights`

### CODE REVIEW

File: src/Instagram.php [hunk 1]
```diff
@@ -268,6 +268,24 @@ public function timelineFeed($maxId = null)
...
+     */
+    public function insights($day = null)
+    {
+        if (empty($day)) {
+            $day = date('d');
+        }
+        $request = $this->request('insights/account_organic_insights')
+        ->addParams('show_promotions_in_landing_page', 'true')
+        ->addParams('first', $day)
+        ->addParams('timezone_name', 'Asia/Jakarta');
```
Comment (kevinjon27) [inline src/Instagram.php]: Sorry, I'm forget about this. You can delete this one

**Comment by mgp25** [comment]
@kevinjon27 `$day` can be null?

**Comment by kevinjon27** [comment]
@mgp25 I will help this project for add some new function if I have free time. Good job with the API!

**Comment by kevinjon27** [comment]
Can anyone help me to see request and response from instagram insights page from see more followers? I cant check because unpinning ssl 😤

**Comment by kevinjon27** [comment]
*pinning ssl

**Comment by mangledbottles** [comment]
@kevinjon27 try something like this https://github.com/ac-pm/SSLUnpinning_Xposed
The idea is to patch the APK

**Pull request merged**